docs/api/createApi.md

```diff
@@ -15,22 +15,90 @@ The main point where you will define a service to use in your application.
 
 ```ts no-transpile
   baseQuery(args: InternalQueryArgs, api: QueryApi): any;
-  entityTypes: readonly EntityTypes[];
-  reducerPath: ReducerPath;
-  serializeQueryArgs?: SerializeQueryArgs<InternalQueryArgs>;
   endpoints(build: EndpointBuilder<InternalQueryArgs, EntityTypes>): Definitions;
+  entityTypes?: readonly EntityTypes[];
+  reducerPath?: ReducerPath;
+  serializeQueryArgs?: SerializeQueryArgs<InternalQueryArgs>;
   keepUnusedDataFor?: number;
 ```
 
+It returns: `reducerPath`, `reducer`, `middleware`, `actions`, `selectors`, `hooks`
+
+```ts
+const api = createApi({
+  baseQuery: fetchBaseQuery('/'),
+  endpoints: (builder) => ({
+    // ...
+  }),
+});
+
+export const { reducerPath, reducer, middleware, hooks, actions, selectors } = api;
+```
+
+#### middleware
+
+This is a standard redux middleware and is responsible for things like [polling](../concepts/polling), [garbage collection](#keepunuseddatafor) and a handful of other things. Make sure it's [included in your store](../introduction/quick-start#add-the-service-to-your-store).
+
+#### actions
+
+React Hooks users will most likely never need to use these in most cases. These are redux action creators that you can `dispatch` with `useDispatch` or `store.dispatch()`.
+
+:::note Usage of actions outside of React Hooks
+When dispatching an action creator, you're responsible for storing a reference to the promise it returns in the event that you want to update that specific subscription. To get an idea of what that entails, see the [Svelte Example](../examples/svelte) or the [React Class Components Example](../examples/react-class-components)
+:::
+
+#### selectors
+
+Selectors are how you access your `query` or `mutation` data from the cache. If you're using Hooks, you don't have to worry about these in most cases. There are several ways to use them:
+
+```js title="React
+// hooks
+const { data, status } = useSelector(selectors.getPosts());
+
+// class component with connect
+const mapStateToProps = (state) => ({
+  data: selectors.getPosts(state),
+});
+connect(null, mapStateToProps);
+```
+
+```js title="Svelte"
+$: ({ data, status } = selectors.getPosts()($store));
+```
+
+#### hooks
+
+Hooks are specifically for React Hooks users. If you are not using React Hooks, you shouldn't `export` them – this way they're not included in the bundle or pull in the React dependency.
```

Comment:
I would remove the reference to "bundle" there. Not exporting them will not reduce bundle size.